general_tools/get_bible_book.py

```diff
@@ -14,74 +14,77 @@
 
 import sys
```

Comment:
Not a requirement, but I would recommend using `from __future__ import unicode_literals`. That way you don't need to put a `u` before every string.